specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,133 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, netmask, status as a part of
+node hardware inventory.
+
+== Use Cases
+
+Tendrl should show the current status of network interface.
+
+== Proposed change
+* Netifaces is used to find the network interface details.
```

Comment:
What is `Netifaces`? Is it a class in the code, a type of object to be used in the flows, some command?